README.md

```diff
@@ -84,18 +84,21 @@ credentials, and enjoy Cylc 8 Alpha-1!
 
 - Start a workflow with the CLI (a good example is shown below)
 - Log in at the Hub to authenticate and launch your UI Server
+![Hub Image](img/hub.png)
```

Comment:
Do you think it would be better to crop the image, removing the browser tabs and window control, focusing only on the Cylc UI page? The rest of the screenshot got more attention from my eyes than the rest of the screenshot 😬 I can quickly crop them with GIMP and send to you if you'd like?

Reply:
You're right - I'll do it.  That's my garish firefox theme :grimacing:

Reply:
Done.